### PR TITLE
Scheduler search fixed

### LINKED ISSUE #3158: Search not working for Scheduler
Search in scheduler does not work.

screenshot from 2017-02-07 10-40-01

### PR DESCRIPTION
**Pull request by magdalenesuo**
Fixed #3158
screenshot from 2017-02-09 21-23-33
screenshot from 2017-02-09 21-23-40

### CODE REVIEW

File: app/static/js/vendor/jquery/jquery.codezero.js [hunk 1]
```diff
@@ -150,13 +150,11 @@ var cachedFuzzyMatch;
...
  * @type {Function}
  */
-if ('undefined' === !typeof _) {
-    cachedFuzzyMatch = _.memoize(function (pattern) {
-        return new RegExp(pattern.split("").reduce(function (a, b) {
-            return a + '[^' + b + ']*' + b;
-        }), 'i');
-    });
-}
+cachedFuzzyMatch = _.memoize(function (pattern) {
```
Comment (niranjan94) [inline app/static/js/vendor/jquery/jquery.codezero.js]: @magdalenesuo the if statement was intentional ... `jquery.codezero.js` is included in some pages where lodash is not included. So in those pages, an error will occur if we try to use a lodash function without the lodash library.

Comment (magdalenesuo) [inline app/static/js/vendor/jquery/jquery.codezero.js]: @niranjan94 That IF function was always returning false though.

Comment (niranjan94) [inline app/static/js/vendor/jquery/jquery.codezero.js]: Huh .. that's weird ... I was the one who had implemented this long ago ... Could you check why ?

Comment (niranjan94) [inline app/static/js/vendor/jquery/jquery.codezero.js]: @magdalenesuo change it to 
`'undefined' !== typeof _`

**Pull request merged**